library/src/main/java/com/novoda/downloadmanager/lib/DownloadsTables.java

```diff
@@ -0,0 +1,18 @@
+package com.novoda.downloadmanager.lib;
+
+final class DownloadsTables {
+
+    /**
+     * Name of downloadstable in the database
+     */
+    public static final String DOWNLOADS_TABLE_NAME = "Downloads";
+    /**
+     * Name of downloadstable in the database
+     */
+    public static final String VIEW_NAME_DOWNLOADS_BY_BATCH = "DownloadsByBatch";
+
+    private DownloadsTables() {
+        // non-instantiable class
+    }
```

Comment:
Should [this](https://github.com/novoda/download-manager/pull/83/files#diff-356b7823c038a7a80d9f6591285507bdR10) be in here as well?